### PR TITLE
Improve configuration properties documentation

### PR DESCRIPTION
**Pull request by majetideepak**
Convert list format to table format.
Resolves https://github.com/facebookincubator/velox/issues/4677

### CODE REVIEW

**Comment by majetideepak** [comment]
File view https://github.com/facebookincubator/velox/blob/d09db6c163ca46599c2d905d32da66d7d384d78f/velox/docs/configs.rst

File: velox/docs/configs.rst [hunk 1]
```diff
@@ -4,230 +4,172 @@ Configuration properties
...
-^^^^^^^^^^^^^^^^^^^^^^^^
-
-    * **Type:** ``bool``
-    * **Default value:** ``false``
-
-True if appending data to an existing unpartitioned table is allowed.
-Currently this configuration does not support appending to existing partitions.
-
+--------------
+``Generic Configuration``
```
Comment (mbasmanova) [inline velox/docs/configs.rst:134]: Should we remove Generic Configuration ?

**Review by mbasmanova — COMMENTED** [review]
@majetideepak Looks good to me % one nit. Thank you.

File: velox/docs/configs.rst (hunk 1 above)
Comment (majetideepak) [reply velox/docs/configs.rst]: Yes, we should. Removed.

**Review by mbasmanova — APPROVED** [review]
@majetideepak Thanks.

**Pull request closed without merging**